TTHAnalysis/python/plotter/susy-sos/2los_cuts.txt

```diff
@@ -6,21 +6,23 @@ eventFilters	: $DATA{EventFilters} $MC{1}	; Disable=True
 dilep		: nLepGood == 2
 sublepPt	: (fabs(LepGood2_pdgId)==13 && LepGood2_pt > 3.5) || (fabs(LepGood2_pdgId)==11 && LepGood2_pt > 5)
 
-mll		: minMllSFOS > 4 && minMllSFOS < 50
-upsilonVeto	: minMllSFOS < 9 || minMllSFOS > 10.5
+mll		: mass_2(LepGood1_pt, LepGood1_eta, LepGood1_phi, LepGood1_mass, LepGood2_pt, LepGood2_eta, LepGood2_phi, LepGood2_mass) > 4 && mass_2(LepGood1_pt, LepGood1_eta, LepGood1_phi, LepGood1_mass, LepGood2_pt, LepGood2_eta, LepGood2_phi, LepGood2_mass) < 50
+upsilonVeto	: mass_2(LepGood1_pt, LepGood1_eta, LepGood1_phi, LepGood1_mass, LepGood2_pt, LepGood2_eta, LepGood2_phi, LepGood2_mass) < 9 || mass_2(LepGood1_pt, LepGood1_eta, LepGood1_phi, LepGood1_mass, LepGood2_pt, LepGood2_eta, LepGood2_phi, LepGood2_mass) > 10.5
 dilepPt		: pt_2(LepGood1_pt, LepGood1_phi, LepGood2_pt, LepGood2_phi) > 3 
 
-ISRjet		: Jet1_jetId >= 4
+ISRjet		: JetSel1_Recl_jetId >= 4
```

Comment:
Since we are substituting this in mcc_sos.txt, wouldn't it be better to use the simpler version here?